sway/input/cursor.c

```diff
@@ -1117,6 +1137,10 @@ struct sway_cursor *sway_cursor_create(struct sway_seat *seat) {
 	wl_signal_add(&wlr_cursor->events.swipe_update, &cursor->swipe_update);
 	cursor->swipe_end.notify = handle_pointer_swipe_end;
 	wl_signal_add(&wlr_cursor->events.swipe_end, &cursor->swipe_end);
+	cursor->hold_begin.notify = handle_pointer_hold_begin;
+	wl_signal_add(&wlr_cursor->events.hold_begin, &cursor->hold_begin);
+	cursor->hold_end.notify = handle_pointer_hold_end;
+	wl_signal_add(&wlr_cursor->events.hold_end, &cursor->hold_end);
```

Comment:
We need to `wl_list_remove` these listeners somewhere.

LGTM apart from this!

Reply:
Fixed :+1: